newrelic-php-daemon/7.6.0.201/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.6
+
+ENV NR_PORT /tmp/.newrelic.sock
+
+ENV NR_VERSION=7.6.0.201
```

Comment:
Let's make the version an ARG with a default so that we can use the same file to build different versions.

Reply:
Done 6d41719